packages/handler-graphql/src/responses.ts

```diff
@@ -37,37 +44,45 @@ export class NotFoundResponse extends ErrorResponse {
 }
 
 export class ListErrorResponse {
-    data: null;
-    meta: null;
-    error: {
+    public readonly data: null;
+    public readonly meta: null;
+    public readonly error: {
         code: string;
         message: string;
         data?: any;
```

Comment:
Let's remove `?` from `data?` (just to be more unified with the above ErrorResponse).